js/pricing.js

```diff
@@ -28,6 +58,67 @@ $(
                 disabledElement.attr('disabled', true);
                 disabledElement.addClass('disabled');
             }
+        };
+
+        /**
+         * Submit order handler.
+         */
+        const submitOrder = () => {
+            const dataProcessingConsent = $confirmPersonalInformation.prop("checked");
+            const supportAgreementConsent = $confirmDevelopmentAgreement.prop("checked");
```

Comment:
Should we add a check to verify that both are `true`? I do not think there is any value in sending a request where the values are `false` right?

And I can do that easily using custom JS.

Also, I guess it makes sense to add the same validation on the backend as well.